src/components/FeatureFlagRenderer.ts

```diff
@@ -23,7 +22,7 @@ export default class FeatureFlagRenderer extends Component<Props, State> {
       checkFeatureFlagComplete: false,
       flagValue: bootstrap &&
         typeof bootstrap === "object" &&
-        bootstrap.hasOwnProperty(flagKey) ? bootstrap[flagKey] : false
+        Object.prototype.hasOwnProperty.call(bootstrap, flagKey) ? bootstrap[flagKey] : false
```

Comment:
Assume you had to change this to get the types working? Are you happy this is well tested, don't understand the full context from just browsing but was just looking for obvious logic changes

Reply:
This is the error that eslint gives me when calling `hasOwnProperty` directly on the object https://eslint.org/docs/rules/no-prototype-builtins. From what they describe there I guess it's safe to call it from prototype